designer/apps/core/templates/wagtailadmin/site_creation.html

```diff
@@ -1,4 +1,4 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "admin/base.html" %}
```

Comment:
I think these paths should stay as they were, was the upgrade utility just too aggressive about renaming things? https://docs.wagtail.io/en/v2.3/advanced_topics/customisation/admin_templates.html

Reply:
reverted back to `wagtailadmin`.